@shared/api/internal.ts

```diff
@@ -310,6 +326,47 @@ export const confirmPassword = async (
   return response;
 };
 
+export const getAccountInfo = async ({
+  publicKey,
+  networkDetails,
+}: {
+  publicKey: string;
+  networkDetails: NetworkDetails;
+}) => {
+  const { networkUrl } = networkDetails;
+
+  const server = new Horizon.Server(networkUrl);
+
+  let account;
+  let signerArr = { records: [] as Horizon.ServerApi.AccountRecord[] };
+
+  try {
+    account = await server.loadAccount(publicKey);
+    signerArr = await server.accounts().forSigner(publicKey).call();
+  } catch (e) {
+    console.error(e);
+  }
+
+  return {
+    account,
+    isSigner: signerArr.records.length > 1,
+  };
```

Comment:
we need some slightly different data than what we store in allAccounts, so we unfortunately need to refetch from Horizon